File: cpp/test/MarsRoverTest.cpp

#include <kata/mars_rover/MarsRover.hpp>

#include <gtest/gtest.h>

using namespace com::murex::kata::mars_rover;

TEST(MarsRover, DISABLED_acceptance_test)
{
	MarsRover rover(10, 10, 'N');
	rover.receiveCommands("LFFLBRRBBBRFFFLL");
	EXPECT_EQ("11 8 W", rover.sendPosition());
}

File: java/src/test/java/com/murex/MarsRoverTest.java
package com.murex;

import org.junit.Ignore;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class MarsRoverTest {

    @Ignore
    @Test
    public void acceptance_test() {
        MarsRover rover = new MarsRover(10, 10, 'N');
        //noinspection SpellCheckingInspection
        rover.receiveCommands("LFFLBRRBBBRFFFLL");
        assertEquals("11 8 W", rover.sendPosition());
    }
}
